src/libratbag-private.h

```diff
@@ -229,6 +248,8 @@ struct ratbag_profile {
 		struct ratbag_resolution *modes;
 		unsigned int num_modes;
 	} resolution;
+	struct ratbag_led *logo_led;
+	struct ratbag_led *side_led;
```

Comment:
This related to what @bentiss said in the bug: it looks like this is a bit too specific and would have to be changed later when we have devices with more/other than logo/side LEDs. I'm ok with it for now since this backend implementation only and can be changed anytime. But I wonder: should we make this an array or linked list from the start?